Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.11
+ADD . /usr/src/nimbess-agent
+RUN mkdir -p $GOPATH/src/github.com/nimbess/nimbess-agent
+WORKDIR $GOPATH/src/github.com/nimbess/nimbess-agent
+COPY . .
+RUN ./build/build-go.sh
+
+FROM alpine:latest
```

Comment:
Note Feng mentioned in the comparable Stargazer patch: eventually switch to a fedora 29 (or whatever we finally get BESS working with) as the final image.

Reply:
will change to centos